test/data/allele_association_ingest_test.json

```diff
@@ -4,7 +4,7 @@
     {
       "curie": "WB:WBVar00000001",
       "allele_symbol_dto": {
-        "name_type_name": "symbol",
+        "name_type_name": "nomenclature_symbol",
```

Comment:
Need to update `inheritance_mode_name` in this test file as `inheritance_mode_dto` now expected